### PR TITLE
Update pre-commit hooks

### PR DESCRIPTION
**Pull request by hartungstenio**
Update pre-commit hooks and replace a deprecated releases by the supported ones.

### CODE REVIEW

**Comment by georgeyk** [comment]
Nice @hartungstenio!
Let me know when you update the branch, I'll prepare a minor release.

**Comment by hartungstenio** [comment]
> Nice @hartungstenio!
> Let me know when you update the branch, I'll prepare a minor release.

@georgeyk Done!

File: loafer/ext/aws/providers.py [hunk 1]
```diff
@@ -3,10 +3,11 @@
 
 import botocore.exceptions
 
-from .bases import BaseSQSClient
 from loafer.exceptions import ProviderError
 from loafer.providers import AbstractProvider
 
+from .bases import BaseSQSClient
```
Comment (hartungstenio) [inline loafer/ext/aws/providers.py:9]: isort complained about this. Maybe I'm missing some setting...

Comment (georgeyk) [reply loafer/ext/aws/providers.py:9]: Try this:
```
sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
no_lines_before=LOCALFOLDER
```

Comment (hartungstenio) [reply loafer/ext/aws/providers.py]: @georgeyk I found it. The setting should be `known_local_folder`.

https://github.com/timothycrosley/isort#custom-sections-and-ordering

**Pull request merged**